src/main/java/com/endava/internship/service/UserServiceImpl.java

```diff
@@ -2,68 +2,99 @@
 
```

Comment:
for streams, try to keep this formatting :
```
list.stream()
    .map(Util::something)
    .collect(toSet())
```

Reply:
each stream function must be started with dot

Reply:
Fixed!